app/views/sign_up/recovery_codes/show.html.slim

```diff
@@ -16,5 +16,14 @@ p.mt-tiny.mb0 = t('instructions.recovery_code')
     data: { print: true },
     class: 'ml2 radio-btn inline-block ico ico-print text-decoration-none'
 
+.mb4.col-12.border.br-md.border-blue.fs-16p.accordion(aria-expanded='true')
+  .accordion-header.py1.px2(aria-controls='recovery-code-info')
+    = t('users.recovery_code.help_text_header')
+  #recovery-code-info.px2.accordion-content.accordion-init.clearfix(aria-hidden='false')
+    p
```

Comment:
I admit, I put the extra P in because adding a `mb1` to the element above it ran afoul of the 100 character line length 😨